# Safe-Haven — a quantitative teardown 🔬
### The inflation correlation · the high/low split · the crash-month behaviour

![Signal: None](https://img.shields.io/badge/Signal-None-c0392b?style=flat-square)
![Tradability: Fragile](https://img.shields.io/badge/Tradability-Fragile-dab617?style=flat-square)
![Gold hedges inflation?: Not supported](https://img.shields.io/badge/Gold_hedges_inflation%3F-Not_supported-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). Gold's two folk jobs, measured — a real diversifier oversold as an inflation hedge and a sure haven.

> ⚠️ **Not investment advice.** GLD / SPY monthly returns + US CPI, 2005–2025 (Yahoo + FRED). Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (safe_haven/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from safe_haven import data, strategy as st
gold_m, eq_ret, cpi_m = data.fetch_panel()     # cache-first (shared pulls)
ih = st.inflation_hedge(gold_m, cpi_m)
cb = st.crisis_ballast(gold_m.pct_change(), eq_ret)


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal (inflation) | **None** | corr +0.09; high−low gap +0.4% |
| Tradability (crisis) | **Fragile** | crash-flat (−0.2% vs −10%) but up 50% |
| Hedges inflation? | **Not supported** | a price bull, not an inflation response |

> 💡 *In plain words:* a diversifier, not the insurance it's sold as.

## 1 · The claims, steelmanned

- **H₁:** YoY gold returns are positively correlated with YoY inflation.
- **H₂:** gold earns more in high-inflation months.
- **H₃:** gold is a safe haven — it rises (not just holds) when equities crash.

## 2 · So what? — what rides on each

H₁/H₂ are the inflation-hedge case; H₃ is the safe-haven case. Failing them doesn't make gold useless — a low/zero correlation is itself valuable for diversification — but it reframes *why* you'd hold it.

## 3 · How we'd know — the protocol

Inflation: corr(YoY gold, YoY CPI) and a high/low (>4%) inflation split. Crisis: corr(gold, stocks) and gold's mean return + up-share in equity-crash months (SPY < −8%).

## 4 · The teardown

### 4.1 The inflation hedge — H₁/H₂

In [2]:
print({k:(round(v,3) if isinstance(v,float) else v) for k,v in ih.items()})

{'corr': 0.093, 'gold_yoy_mean': 0.102, 'infl_yoy_mean': 0.026, 'gold_hi_infl': 0.106, 'gold_lo_infl': 0.102, 'hi_minus_lo': 0.004, 'n': 231, 'n_hi': 38}


> 💡 *In plain words:* corr +0.09, and high-inflation gold (+10.6%) ≈ low-inflation gold (+10.2%). **H₁ and H₂ rejected** — no month-to-month inflation hedge (Erb-Harvey 2013).

### 4.2 The safe haven — H₃

In [3]:
print({k:(round(v,3) if isinstance(v,float) else v) for k,v in cb.items()})

{'stock_corr': 0.084, 'gold_in_crash': -0.002, 'eq_in_crash': -0.101, 'gold_up_share': 0.5, 'n_crash': 10, 'n': 259}


> 💡 *In plain words:* gold is near-uncorrelated with stocks (+0.08) and *flat* in the 10 worst equity months (−0.2% vs −10.1%) — but it rose in only **50%** of them. **H₃ partly rejected:** gold is a *hedge* (uncorrelated ballast) but not a reliable *safe haven* (Baur-Lucey 2010's distinction).

## 5 · The verdict

H₁/H₂ rejected, H₃ partly rejected → Signal `NONE` (inflation), Tradability `FRAGILE` (crisis), hedges-inflation `NOT SUPPORTED`. The fingerprinted run is in [`docs/results.md`](../docs/results.md).

## 6 · Could you trade it?

Hold gold for *diversification* — the +0.08 stock correlation and crash-flat behaviour are genuinely useful in a portfolio (it earns its place in risk parity, [68](../../68-all-weather/)). But size it as a diversifier, not as inflation insurance or guaranteed crisis alpha.

## 7 · Going further

Forks: (a) the very-long-horizon real gold price (Erb-Harvey's mean-reversion); (b) gold vs TIPS as the real inflation hedge; (c) real-rate and dollar drivers; (d) safe-haven behaviour conditioned on crisis *type*. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).